es-space-mgmt/main/default/lwc/relatedSpaces/__tests__/relatedSpaces.test.js

```diff
@@ -0,0 +1,136 @@
+import { createElement } from 'lwc';
+import getRelatedSpaces from '@salesforce/apex/marketServices.getRelatedSpaces';
+import { getNavigateCalledWith } from 'lightning/navigation';
+import { registerApexTestWireAdapter } from '@salesforce/sfdx-lwc-jest';
+import RelatedSpaces from 'c/relatedSpaces';
+
+// Realistic data with a list of spaces
+const mockRelatedSpaceRecords = require('./data/getRelatedSpaces.json');
+
+// An empty list of records to verify the component does something reasonable
+// when there is no data to display
+const mockRelatedSpacesNoRecords = require('./data/getRelatedSpacesNoRecords.json');
+
+// Register as Apex wire adapter. Some tests verify that provisioned values trigger desired behavior.
+const getRelatedSpacesAdapter = registerApexTestWireAdapter(getRelatedSpaces);
+
+describe('c-related-spaces', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+        // Prevent data saved on mocks from leaking between tests
+        jest.clearAllMocks();
+    });
+
+    it('getRelatedSpaces @wire data', () => {
+        const RECORDID = '0031700000pJRRSAA4';
```

Comment:
No biggie, just that "recordId" are two words `RECORD_ID` would make more sense.